src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -137,53 +137,20 @@ public EntryEditor(BasePanel panel, ExternalFileTypes externalFileTypes) {
                 FileDragDropPreferenceType dragDropPreferencesType = preferencesService.getEntryEditorFileLinkPreference();
 
                 if (dragDropPreferencesType == FileDragDropPreferenceType.MOVE) {
```

Comment:
Can we change this to a `switch` statement?

Reply:
I actually found the old code confusing. If I select the default action to be linking and not moving, I would expect that behavior without pressing some modifier key.
However, if we decide to use modifier keys we should remove this setting from the preferences as the user gets the impression it is ignored. See the report in the linked issue.

My personal opinion would be to remove this modifier stuff at all. I doubt that the normal user is aware of the modifier keys

Checking for the OS does not make any sense, as it for linux depends on the desktop system the user has, and this might be different behaviors or not supporting modifiers

Reply:
I checked https://github.com/JabRef/jabref/issues/5546. There, it is only talked about WITHOUT modifier keys.

As professional Windows user, I use modifier keys, too. For me, it is OK, if modifiers worked on Windows only and had no effect on other operating systems.

Since no one else seems to know that one can press modifier keys while copying, we can let this in and in a follow-up issue discuss and implement a proper solution.